engine/polyglot-api/src/main/scala/org/enso/polyglot/ExportedSymbol.scala

```diff
@@ -1,28 +1,7 @@
 package org.enso.polyglot
 
-import com.fasterxml.jackson.annotation.{JsonSubTypes, JsonTypeInfo}
```

Comment:
Removing Jackson import is good. Do we also remove the Jackson dependency?

Reply:
I did, see `build.sbt`.